kv/raftstore/peer.go

```diff
@@ -206,12 +202,6 @@ func (p *peer) MaybeDestroy() bool {
 		log.Infof("%v is being destroyed, skip", p.Tag)
 		return false
 	}
-	if p.IsApplyingSnapshot() {
-		if !p.Store().CancelApplyingSnap() {
```

Comment:
please keep this now, it's just a place holder. May finish the implementation later